trade-finance-manager-ui/test-helpers/test-data/add-payment-view-model.ts

```diff
@@ -0,0 +1,10 @@
+import { AddPaymentViewModel } from '../../server/types/view-models';
+
+export const anAddPaymentViewModel = (): AddPaymentViewModel => ({
+  bank: { name: 'Test bank ' },
+  reportPeriod: 'Some reporting period',
+  reportedFeeDetails: {
+    totalReportedPayments: 'GBP 200',
+    feeRecords: [],
```

Comment:
Shouldn't matter as this is just mock data, but do you think we could get any unexpected results from having a value for `totalReportedPayments` when `feeRecords` is empty?

Reply:
I do think mock data should be a representation of a realistic state so I will add in one, I think it just felt like effort at the time when this did satisfy the type 😅